mbof/views.py

```diff
@@ -28,9 +29,24 @@ def log_action(request):
     # EXAMPLE - dovek : GET /api/events/current/
     message = '%s : %s %s' % (me, method, url)
     logger.info(message)
-    
 
-class UserViewSet(viewsets.ModelViewSet):
+
+class OcellusViewSet(mixins.CreateModelMixin,
+                       mixins.RetrieveModelMixin,
+                       mixins.UpdateModelMixin,
+                       # mixins.DestroyModelMixin,
+                       mixins.ListModelMixin,
+                       viewsets.GenericViewSet):
+
+
+    """
+    A viewset that provides default `create()`, `retrieve()`, `update()`,
+    `partial_update()`, `destroy()` and `list()` actions.
```

Comment:
Does it **_really_** provide the `destroy()` method?  Misleading comments are worse than no comment at all.